index.js

```diff
@@ -82,6 +84,14 @@ class ServerlessCustomDomain {
     this.targetHostedZoneName = this.givenDomainName.substring(this.givenDomainName.indexOf('.') + 1);
   }
 
+  setEndpointType(endpointType) {
+    this.endpointType = endpointType || 'EDGE';
```

Comment:
Maybe should validate the value of `endpointType` somehow to make sure its either `EDGE` or `REGIONAL`